### PR TITLE
XSOAR [CSPi184] Aria packet intelligence update with latest REST API

### PR DESCRIPTION
**Pull request by stang-aria**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
This update using the latest REST API of ARIA PacketIntelligence. Update the existing Integration ARIAPacketIntelligence.
Use RCS arguments to select different SIA.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [x] 6.0.0

## Does it break backward compatibility?
   - [x] Yes
       - Further details: minimum SDS version 1.0.9
   - [ ] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by stang-aria** [comment]
There are two possible issues for Run Validations from CI/CD testing. I have a problem to commit the ARIAPacketIntelligence.yml file because the demisto-sdk has a pre-check and give me this error messages. Our latest updates will break the backward compatibility, is there anyway to skip this?
`================= Running validation on modified files =================

Validating Packs/ARIAPacketIntelligence/Integrations/ARIAPacketIntelligence/ARIAPacketIntelligence.yml as integration
Validating scheme for Packs/ARIAPacketIntelligence/Integrations/ARIAPacketIntelligence/ARIAPacketIntelligence.yml
Packs/ARIAPacketIntelligence/Integrations/ARIAPacketIntelligence/ARIAPacketIntelligence.yml: [BC104] - Possible backwards compatibility break, You've changed the name of a command or its arg in the file, please undo, the command was:
aria-block-conversation
`
And the failure of parsing the README.md, seems like it is a existing CI/CD issue I have met before

File: Packs/ARIAPacketIntelligence/pack_metadata.json [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-  "created": "2020-04-14T00:00:00Z",
-  "categories": [
-    "Network Security"
-  ],
-  "tags": [],
-  "useCases": [],
-  "keywords": []
+    "name": "ARIAPacketIntelligence",
+    "description": "Manage Packet Intelligence rules in response to incidents. Instantly block conversations, redirect packets, generate alerts, or perform other actions.",
+    "support": "xsoar",
```
Comment (bakatzir) [inline Packs/ARIAPacketIntelligence/pack_metadata.json:4]:
```suggestion
    "support": "partner",
```

File: Packs/ARIAPacketIntelligence/pack_metadata.json [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+    "currentVersion": "2.0.0",
+    "author": "ARIA Cybersecurity Solutions",
+    "url": "https://www.ariacybersecurity.com/cybersecurity-products/aria-packet-intelligence",
+    "email": "ARIA_support@ariacybersecurity.com",
+    "created": "2020-09-17T22:12:55Z",
+    "categories": [
+        "Network Security"
+    ],
+    "tags": ["Alerts", "Incident Response", "Network", "Security", "Threat Intelligence"],
+    "useCases": ["Incident Response", "Threat Intelligence Management"],
```
Comment (DeanArbel) [inline Packs/ARIAPacketIntelligence/pack_metadata.json:14]:
```suggestion
    "useCases": ["Incident Response", "Threat Intelligence Management"],
    "certification": "certified",
```

File: Packs/ARIAPacketIntelligence/Integrations/ARIAPacketIntelligence/README.md [hunk 3]
```diff
@@ -1,2188 +1,2311 @@
...
 ```
 
-##### Human Readable Output
+#### Human Readable Output
 
-##### aria-block-conversation
+>### aria-block-conversation
+>|Rule|Status|Endpoints|
+>|---|---|---|
+>| Name: convBlock<br/>Definition: 192.168.0.1/32 @ 390 & 192.168.10.23/32 @ 389 <> TCP : DROP, END<br/>RCS: PIdevice@all | command_state: Success<br/>timestamp: 1601688609 | {'FQN': '<sds_cluster_0>.<sds_node_sia12>.<sds_component_PacketIntelligence>.<sds_uuid_07023d45-d4a0-4204-949d-86ce009fd172>', 'IPAddress': '192.168.0.100', 'Model': 'sia-lx2160', 'OS': 'GNU/Linux', 'Processor': 'sia-lx2160', 'trid': '230e10c2-0dea-c12f-8929-092130038061', 'instance_number': '0', 'completion': True},<br/>{'FQN': '<sds_cluster_0>.<sds_node_sia32>.<sds_component_PacketIntelligence>.<sds_uuid_590c49ce-1286-481b-ae07-f4192130e7af>', 'IPAddress': '192.168.0.101', 'Model': 'sia-lx2160', 'OS': 'GNU/Linux', 'Processor': 'sia-lx2160', 'trid': '04972eb4-45b2-a877-12f2-4fcc7638f6c1', 'instance_number': '0', 'completion': True} |
```
Comment (DeanArbel) [inline Packs/ARIAPacketIntelligence/Integrations/ARIAPacketIntelligence/README.md:102]: Please use <br/> and not <br> in README.md

**Review by DeanArbel — CHANGES_REQUESTED** [review]
Awesome work on this pull request!
- I see you provided a test playbook, but I see it's not configured in `conf.json`. Is it possible to receive a license so it could be tested in the content build?
- See the rest of my comments.

File: Packs/ARIAPacketIntelligence/Integrations/ARIAPacketIntelligence/ARIAPacketIntelligence.yml [hunk 4]
```diff
@@ -1009,24 +893,20 @@ script:
...
     - contextPath: Aria.MuteAlertSrcSubnet.Status
-      description: 'The response code, the state of the command, and the timestamp indicating when the command completed. If an error occurs or the response code is not 201, this also returns information about the error.'
+      description: 'The state of the command, and the timestamp indicating when the command completed. Possible states include "Success", "Failure", or "Endpoint matching RCS not found".'
       type: string
     - contextPath: Aria.MuteAlertSrcSubnet.Endpoints
       description: 'Endpoint information, such as the IP address, about the SIAs that were modified based on the rule change.'
       type: string
     description: Removes a named rule from the source subnet logic block, disabling the alerts.
-  dockerimage: demisto/python3:3.7.4.1502
+  dockerimage: demisto/python3:3.8.6.12176
```
Comment (DeanArbel) [inline Packs/ARIAPacketIntelligence/Integrations/ARIAPacketIntelligence/ARIAPacketIntelligence.yml:909]:
```suggestion
  dockerimage: demisto/python3:3.8.6.13358
```

**Comment by DeanArbel** [comment]
> There are two possible issues for Run Validations from CI/CD testing. I have a problem to commit the ARIAPacketIntelligence.yml file because the demisto-sdk has a pre-check and give me this error messages. Our latest updates will break the backward compatibility, is there anyway to skip this?

@stang-aria If you believe this is not a break of backward compatibility, or that the changes are required, then please push it without a pre-check, and comment about it. If needed, we'll force merge the PR. We might need to add a warning to the release notes of this change detailing in what way backward compatibility was broken.

> And the failure of parsing the README.md, seems like it is a existing CI/CD issue I have met before

Feel free to ignore this error.

**Comment by stang-aria** [comment]
@DeanArbel Thanks very much for your review. I pushed my changes without pre-commit by removing the hooks to skip the demisto-sdk validation. Is there any better ways to do this? How to add a warning to the release notes of this change detailing in what way backward compatibility was broken? The pack_metadata.json and the doc-string/comments should be resolved.

**Comment by stang-aria** [comment]
@DeanArbel right now there is only a CI/CD issue in README.md and I think that can be ignored. Just let me know if you have any more comments here and when can you make the push. Very appreciate!

File: Packs/ARIAPacketIntelligence/Integrations/ARIAPacketIntelligence/README.md (hunk 3 above)
Comment (DeanArbel) [inline Packs/ARIAPacketIntelligence/Integrations/ARIAPacketIntelligence/README.md:102]: Please add an escape character to the `<>` characters.
```suggestion
>| Name: convBlock<br/>Definition: 192.168.0.1/32 @ 390 & 192.168.10.23/32 @ 389 \<\> TCP : DROP, END<br/>RCS: PIdevice@all | command_state: Success<br/>timestamp: 1601688609 | {'FQN': '<sds_cluster_0>.<sds_node_sia12>.<sds_component_PacketIntelligence>.<sds_uuid_07023d45-d4a0-4204-949d-86ce009fd172>', 'IPAddress': '192.168.0.100', 'Model': 'sia-lx2160', 'OS': 'GNU/Linux', 'Processor': 'sia-lx2160', 'trid': '230e10c2-0dea-c12f-8929-092130038061', 'instance_number': '0', 'completion': True},<br/>{'FQN': '<sds_cluster_0>.<sds_node_sia32>.<sds_component_PacketIntelligence>.<sds_uuid_590c49ce-1286-481b-ae07-f4192130e7af>', 'IPAddress': '192.168.0.101', 'Model': 'sia-lx2160', 'OS': 'GNU/Linux', 'Processor': 'sia-lx2160', 'trid': '04972eb4-45b2-a877-12f2-4fcc7638f6c1', 'instance_number': '0', 'completion': True} |
```

File: Packs/ARIAPacketIntelligence/Integrations/ARIAPacketIntelligence/README.md [hunk 5]
```diff
@@ -1,2188 +1,2311 @@
...
+>### aria-record-conversation
+>|Rule|Status|Endpoints|
+>|---|---|---|
+>| Name: convRecord<br/>Definition: 192.168.0.1/32 @ 390 & 192.168.10.23/32 @ 389 <> TCP : REDIRECT-VLAN A 1234, ALERT email 2 4 one-shot 1, END<br/>RCS: PIdevice@all | command_state: Success<br/>timestamp: 1601688621 | {'FQN': '<sds_cluster_0>.<sds_node_sia32>.<sds_component_PacketIntelligence>.<sds_uuid_590c49ce-1286-481b-ae07-f4192130e7af>', 'IPAddress': '192.168.0.101', 'Model': 'sia-lx2160', 'OS': 'GNU/Linux', 'Processor': 'sia-lx2160', 'trid': '0fbc758b-ab0d-95f4-9955-56c96089fa98', 'instance_number': '0', 'completion': True},<br/>{'FQN': '<sds_cluster_0>.<sds_node_sia12>.<sds_component_PacketIntelligence>.<sds_uuid_07023d45-d4a0-4204-949d-86ce009fd172>', 'IPAddress': '192.168.0.100', 'Model': 'sia-lx2160', 'OS': 'GNU/Linux', 'Processor': 'sia-lx2160', 'trid': 'ab63e403-4f3a-1f3c-6005-c01d249eb185', 'instance_number': '0', 'completion': True} |
```
Comment (DeanArbel) [inline Packs/ARIAPacketIntelligence/Integrations/ARIAPacketIntelligence/README.md:266]: Please add an escape character to the <> characters.
```suggestion
>| Name: convRecord<br/>Definition: 192.168.0.1/32 @ 390 & 192.168.10.23/32 @ 389 \<\> TCP : REDIRECT-VLAN A 1234, ALERT email 2 4 one-shot 1, END<br/>RCS: PIdevice@all | command_state: Success<br/>timestamp: 1601688621 | {'FQN': '<sds_cluster_0>.<sds_node_sia32>.<sds_component_PacketIntelligence>.<sds_uuid_590c49ce-1286-481b-ae07-f4192130e7af>', 'IPAddress': '192.168.0.101', 'Model': 'sia-lx2160', 'OS': 'GNU/Linux', 'Processor': 'sia-lx2160', 'trid': '0fbc758b-ab0d-95f4-9955-56c96089fa98', 'instance_number': '0', 'completion': True},<br/>{'FQN': '<sds_cluster_0>.<sds_node_sia12>.<sds_component_PacketIntelligence>.<sds_uuid_07023d45-d4a0-4204-949d-86ce009fd172>', 'IPAddress': '192.168.0.100', 'Model': 'sia-lx2160', 'OS': 'GNU/Linux', 'Processor': 'sia-lx2160', 'trid': 'ab63e403-4f3a-1f3c-6005-c01d249eb185', 'instance_number': '0', 'completion': True} |
```

File: Packs/ARIAPacketIntelligence/Integrations/ARIAPacketIntelligence/README.md [hunk 6]
```diff
@@ -1,2188 +1,2311 @@
...
+>### aria-alert-conversation
+>|Rule|Status|Endpoints|
+>|---|---|---|
+>| Name: convAlert<br/>Definition: 192.168.0.1/32 @ 390 & 192.168.10.23/32 @ 389 <> TCP : ALERT email 2 4 re-trigger-count 1000, END<br/>RCS: null | command_state: Success<br/>timestamp: 1601688638 | {'FQN': '<sds_cluster_0>.<sds_node_sia32>.<sds_component_PacketIntelligence>.<sds_uuid_590c49ce-1286-481b-ae07-f4192130e7af>', 'IPAddress': '192.168.0.101', 'Model': 'sia-lx2160', 'OS': 'GNU/Linux', 'Processor': 'sia-lx2160', 'trid': '92573cdb-9e81-b408-1417-6668cddae433', 'instance_number': '0', 'completion': True},<br/>{'FQN': '<sds_cluster_0>.<sds_node_sia12>.<sds_component_PacketIntelligence>.<sds_uuid_07023d45-d4a0-4204-949d-86ce009fd172>', 'IPAddress': '192.168.0.100', 'Model': 'sia-lx2160', 'OS': 'GNU/Linux', 'Processor': 'sia-lx2160', 'trid': '48808901-91b2-5284-d1a3-39cb5fcb1add', 'instance_number': '0', 'completion': True} |
```
Comment (DeanArbel) [inline Packs/ARIAPacketIntelligence/Integrations/ARIAPacketIntelligence/README.md:428]: Please add an escape character to the <> characters.
```suggestion
>| Name: convAlert<br/>Definition: 192.168.0.1/32 @ 390 & 192.168.10.23/32 @ 389 \<\> TCP : ALERT email 2 4 re-trigger-count 1000, END<br/>RCS: null | command_state: Success<br/>timestamp: 1601688638 | {'FQN': '<sds_cluster_0>.<sds_node_sia32>.<sds_component_PacketIntelligence>.<sds_uuid_590c49ce-1286-481b-ae07-f4192130e7af>', 'IPAddress': '192.168.0.101', 'Model': 'sia-lx2160', 'OS': 'GNU/Linux', 'Processor': 'sia-lx2160', 'trid': '92573cdb-9e81-b408-1417-6668cddae433', 'instance_number': '0', 'completion': True},<br/>{'FQN': '<sds_cluster_0>.<sds_node_sia12>.<sds_component_PacketIntelligence>.<sds_uuid_07023d45-d4a0-4204-949d-86ce009fd172>', 'IPAddress': '192.168.0.100', 'Model': 'sia-lx2160', 'OS': 'GNU/Linux', 'Processor': 'sia-lx2160', 'trid': '48808901-91b2-5284-d1a3-39cb5fcb1add', 'instance_number': '0', 'completion': True} |
```

**Review by DeanArbel — CHANGES_REQUESTED** [review]
@stang-aria The ci errors you're experiencing are related to the way the `md` file is parsed. It tries to translate the table HTML elements to the appropriate HTML representation.
In this case, it sees `<>` as an HTML element, and fails to find a closure to this element.

Since this is not actually an HTML element, you should add escape characters to the brackets - see the suggestions below.

**Comment by stang-aria** [comment]
@DeanArbel Thanks for your comments. Fixed for this issue.

**Review by DeanArbel — APPROVED** [review]

**Pull request merged**